#!/bin/sh -e
## 50_gcc4.dpatch by Andreas Jochens
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Get source building with GCC 4  (bug #287923)

if [ $# -lt 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
       -patch) patch $patch_opts -p0 < $0;;
       -unpatch) patch $patch_opts -p0 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0

@DPATCH@
--- Makefile.inc.orig	2005-10-24 00:46:46.000000000 +0200
+++ Makefile.inc	2005-10-24 00:50:38.000000000 +0200
@@ -54,9 +54,9 @@
 MAKEFLAGS   += -s
 VERSION      = `$(TOPDIR)/test/xtract_version`
 INST_WORKDIR = $(DESTDIR)'/var/lib/dibbler'
-INST_MANDIR  = $(DESTDIR)'/usr/share/man'
-INST_DOCDIR  = $(DESTDIR)'/usr/share/doc'
-INST_BINDIR  = $(DESTDIR)'/usr/sbin'
+INST_MANDIR  = $(DESTDIR)'/usr/local/man'
+INST_DOCDIR  = $(DESTDIR)'/usr/local/share/doc'
+INST_BINDIR  = $(DESTDIR)'/usr/local/sbin'
 INST_CONFDIR = $(DESTDIR)'/etc/dibbler'
 INSTALL      = 'install'
 MKDIR        = mkdir -p
